File: backend/src/main/java/ch/fhnw/labeling_tool/model/TextAudioDto.java
package ch.fhnw.labeling_tool.model;

public class TextAudioDto {
    public final Long id;
    public final Double audioStart;
    public final Double audioEnd;
    public final String text;

    public TextAudioDto(Long id, Double audioStart, Double audioEnd, String text) {
        this.id = id;
        this.audioStart = audioStart;
        this.audioEnd = audioEnd;
        this.text = text;
    }

}

File: frontend/src/app/models/text-audio-dto.ts
export class TextAudioDto {
  id: number;
  audioStart: number;
  audioEnd: number;
  text: string;

  constructor(id: number, audioStart: number, audioEnd: number, text: string) {
    this.id = id;
    this.audioStart = audioStart;
    this.audioEnd = audioEnd;
    this.text = text;
  }
}
